kitsune/sumo/templatetags/jinja_helpers.py

```diff
@@ -129,7 +129,13 @@ def wiki_to_html(
 @library.filter
 def wiki_to_safe_html(wiki_markup, locale=settings.WIKI_DEFAULT_LANGUAGE, nofollow=True):
     """Wiki Markup -> HTML Markup object with limited tags"""
-    html = parser.wiki_to_html(wiki_markup, locale=locale, nofollow=nofollow)
+    html = parser.wiki_to_html(
+        wiki_markup,
+        locale=locale,
+        nofollow=nofollow,
+        tags=ALLOWED_BIO_TAGS,
+        attributes=ALLOWED_BIO_ATTRIBUTES,
```

Comment:
Hm, these changes are causing the`kitsune.users.tests.test_templates.ViewProfileTests.test_bio_links_no_img` test to fail, but something like them is needed for the `<abbr>` tag to work. Going to dig into this further.